src/styles/main.scss

```diff
@@ -1,3 +1,36 @@
 body {
   margin: 0;
 }
+
+$colors:
+  "1" hsl(0, 100%, 50%),
+  "2" hsl(0, 100%, 40%),
+  "3" hsl(0, 100%, 30%),
+  "4" hsl(0, 100%, 20%),
+  "5" hsl(0, 100%, 10%),
+  "6" #000;
+
+.container {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+
+  @media (min-width:1200px) {
+    grid-template-columns: 1fr 1fr 1fr;
+  }
+
+  &__block {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 300px;
+    font-size: 100px;
+    color: white;
+    font-family: Arial, Helvetica, sans-serif;
+
+    @each $name,$color in $colors {
+      &--#{$name} {
+        background-color: $color;
+      }
+    }
```

Comment:
Don't create any extra classes, try to implement this without creating any extra classes.

*Hint:* use `nth-child` pseudoclass.